File: management_app/lib/models/user.model.dart
class User {
  var id = 0;
  var name = "";
  var role = "";
  var email = "";
  var password = "";
  User({
    required this.id,
    required this.name,
    this.role = "member",
    required this.email,
    required this.password,
  });
}

File: management_apis/models/user.model.go
package models

type UserRegisRequest struct {
	Email    string `json:"email" validate:"required,email"`
	Password string `json:"password" validate:"required,min=6,max=50"`
	FullName string `json:"full_name" validate:"required"`
	Roles    string `json:"roles" validate:"required"`
}
